client/landing/stepper/declarative-flow/internals/steps-repository/newsletter-setup/style.scss

```
@@ -105,3 +105,9 @@ $border-radius: 4px;
 		width: unset !important;
 	}
 }
+
+.newsletter-setup__accent-color-popover {
+	&:focus {
```

Optional: Let's add this because it still reproduces the issue.

```suggestion
	&:focus,
	&:focus-visible {
```

---

sure 👍